core/jazz_acl/index.js

```diff
@@ -24,55 +23,94 @@ const casbinUtil = require("./components/casbin.js");
 module.exports.handler = async(event, context) => {
 
   //Initializations
-  const errorHandler = errorHandlerModule();
   const config = configModule.getConfig(event, context);
   logger.init(event, context);
 
   try {
 
-    await validation.validateInput(event);
-    const aclResult = await processACL(event);
-
-    return responseObj(aclResult, event.body);
+    validation.validateBasicInput(event);
+    const aclResult = await processACLRequest(event, config);
 
+    return {
+      status: 200,
+      body: aclResult
```

Comment:
- `status` is not required. 
-  key `body` doesn't exist in swagger. Are we sending payload as per the swagger spec?

Reply:
updated